File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacModel/EliminableDTOBase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftPac.Model
{
    public abstract class EliminableDTOBase
    {
        private UsuariosDTO usuarioEliminacion;
        private DateTime? fechaEliminacion;

        public EliminableDTOBase()
        {
            this.UsuarioEliminacion = null;
            this.FechaEliminacion = null; 
        }

        public EliminableDTOBase(EliminableDTOBase other)
        {
            this.UsuarioEliminacion = other.UsuarioEliminacion;
            this.FechaEliminacion = other.FechaEliminacion;
        }

        public UsuariosDTO UsuarioEliminacion { get => usuarioEliminacion; set => usuarioEliminacion = value; }
        public DateTime? FechaEliminacion { get => fechaEliminacion; set => fechaEliminacion = value; }
    }
}

File: BackEnd/SoftPacModel/src/main/java/pe/edu/pucp/softpac/model/EliminableDTOBase.java

package pe.edu.pucp.softpac.model;

import java.util.Date;

public abstract class EliminableDTOBase {
    
    private UsuariosDTO usuario_eliminacion;
    private Date fecha_eliminacion;
    
    public EliminableDTOBase(){
        usuario_eliminacion = null;
        fecha_eliminacion = null;
    }
    
    public EliminableDTOBase(EliminableDTOBase eliminable){
        this.fecha_eliminacion = eliminable.getFecha_eliminacion();
        this.usuario_eliminacion = eliminable.getUsuario_eliminacion();
    }
    
    /**
     * @return the usuario_eliminacion
     */
    public UsuariosDTO getUsuario_eliminacion() {
        return usuario_eliminacion;
    }

    /**
     * @param usuario_eliminacion the usuario_eliminacion to set
     */
    public void setUsuario_eliminacion(UsuariosDTO usuario_eliminacion) {
        this.usuario_eliminacion = usuario_eliminacion;
    }

    /**
     * @return the fecha_eliminacion
     */
    public Date getFecha_eliminacion() {
        return fecha_eliminacion;
    }

    /**
     * @param fecha_eliminacion the fecha_eliminacion to set
     */
    public void setFecha_eliminacion(Date fecha_eliminacion) {
        this.fecha_eliminacion = fecha_eliminacion;
    }
     
    
}
